cf_xarray/tests/test_accessor.py

```diff
@@ -340,19 +340,18 @@ def test_getitem_ancillary_variables():
 
 
 def test_rename_like():
-    original = popds.copy(deep=True)
-
     # it'll match for axis: X (lon, nlon) and coordinate="longitude" (lon, TLONG)
     # so delete the axis attributes
-    newair = airds.copy(deep=True)
+    original = popds
+    newair = airds.copy(deep=False)
     del newair.lon.attrs["axis"]
     del newair.lat.attrs["axis"]
 
     renamed = popds.cf["TEMP"].cf.rename_like(newair)
+    assert original.TEMP.attrs["coordinates"] == "TLONG TLAT"
     for k in ["TLONG", "TLAT"]:
-        assert k not in renamed.coords
         assert k in original.coords
-    assert original.TEMP.attrs["coordinates"] == "TLONG TLAT"
+        assert k not in renamed.coords
```

Comment:
Nothing changed here. Correct?

Reply:
Yep that's right. Didn't change it for any particularly big reason.